src/web-ui/src/index.css

```diff
@@ -339,5 +339,4 @@ svg {
 
 .amplify-auth-container {
   text-align: center;
-  margin: 5% auto 50px;
```

Comment:
Did you mean to include this in the PR?

Reply:
Yes, by including the latest version of the UI components for react, I noticed that this hack (which was basically to center the Login window in the screen) wasn't needed anymore as the window is now well centered by default (and with this code, we have an extra 50px that we don't need anymore).